components/browser/engine-system/src/test/java/mozilla/components/browser/engine/system/SystemEngineSessionTest.kt

```diff
@@ -379,7 +379,6 @@ class SystemEngineSessionTest {
         verify(webViewSettings).userAgentString = "userAgent"
         verify(webViewSettings).mediaPlaybackRequiresUserGesture = false
         verify(webViewSettings).javaScriptCanOpenWindowsAutomatically = true
-        verify(webViewSettings).displayZoomControls = false
```

Comment:
We definitely want to keep this in our tests. Maybe there was something else that needed to be altered in the test?

Reply:
Well, without this, it passes fine.